client/src/modules/stock/exit/exit.js

```diff
@@ -32,6 +32,7 @@ function StockExitController(
   vm.currentInventories = [];
   vm.reset = reset;
   vm.ROW_ERROR_FLAG = bhConstants.grid.ROW_ERROR_FLAG;
+  vm.DATE_FMT = bhConstants.dates.format;
```

Comment:
This is fine for now, while we have Bhima installed in only one country.  For the future we should think about allowing the admin to select the desired date format (or use default regional formats).

Reply:
> This is fine for now, while we have Bhima installed in only one country. For the future we should think about allowing the admin to select the desired date format (or use default regional formats).

@jniles Would you like me to add an issue for this?